src/Sylius/Bundle/OrderBundle/Controller/AdjustmentController.php

```diff
@@ -53,4 +53,33 @@ public function lockAction(Request $request, $id)
 
         return $this->redirectHandler->redirectTo($order);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function createNew()
+    {
+        if (null === $orderId = $this->getRequest()->get('orderId')) {
+            throw new NotFoundHttpException('No order id given.');
```

Comment:
Actually this should throw 400 bad request.

:+1: for createNotFound in condition below